### PR TITLE
Use the default_theme value from the login

### LINKED ISSUE #654: Set default theme in mobile / desktop apps
Spec: https://github.com/camptocamp/c2cgeoportal/wiki/Spec-%23654-Functionalities-in-the-ngeo-application

Ideally, the default theme should come from the `themes` service

### PR DESCRIPTION
**Pull request by fredj**
fixes #654

### CODE REVIEW

File: contribs/gmf/src/directives/themeselector.js [hunk 1]
```diff
@@ -165,25 +172,54 @@ gmf.ThemeselectorController.prototype.setThemes_ = function() {
...
+    //   }
+    // }
+    // currentTheme = gmf.Themes.findThemeByName(this.themes, themeName);
+    // if (currentTheme === null) {
+    //   // if the current theme is not available (e.g. a restricted theme is no
+    //   // longer available after a logout), use the default
+    //   currentTheme = gmf.Themes.findThemeByName(this.themes, this.defaultTheme);
+    // }
+
+    // this.setTheme(/** @type {GmfThemesNode} */ (currentTheme), true);
```
Comment (sbrunner) [inline contribs/gmf/src/directives/themeselector.js]: Shouldn't we remove this commented code?

Comment (fredj) [inline contribs/gmf/src/directives/themeselector.js]: still in "work in progress" ...

Comment (sbrunner) [inline contribs/gmf/src/directives/themeselector.js]: ok :-)

**Comment by fredj** [comment]
It can't be tested because this pull request depends on https://github.com/camptocamp/c2cgeoportal/pull/2456 and geomapfish-demo.camptocamp.net is not yet updated

**Comment by sbrunner** [comment]
Demo updated :-)

**Comment by fredj** [comment]
@sbrunner still no `default_theme` in https://geomapfish-demo.camptocamp.net/2.1/wsgi/loginuser (admin or anonymous)

**Comment by fredj** [comment]
But we can merge and check later with the demo

**Comment by fredj** [comment]
@sbrunner ready to be reviewed

**Comment by fredj** [comment]
ping @sbrunner

**Comment by sbrunner** [comment]
For me that's looks good :-)

**Comment by fredj** [comment]
Thanks for the review

**Pull request merged**